contributing/style_guide.md

```diff
@@ -231,7 +231,7 @@ export function foo() {
 }
 ```
 
-It is important that documentation is easily human readable, but there is also a
+It is important that documentation is easily human-readable, but there is also a
```

Comment:
Compound noun modifiers should be hyphenated.
Eg: `human-readable documentation`